runelite-client/src/main/java/net/runelite/client/plugins/tearsofguthix/TearsOfGuthixPlugin.java

```diff
@@ -47,7 +54,14 @@
 )
 public class TearsOfGuthixPlugin extends Plugin
 {
-	private static final int TOG_REGION = 12948;
+	static final int TOG_REGION = 12948;
+	private static final Set<Integer> TEARS_OBJECTS = new ImmutableSet.Builder<Integer>().
+			add(ObjectID.BLUE_TEARS).
+			add(ObjectID.BLUE_TEARS_6665).
+			add(ObjectID.GREEN_TEARS).
+			add(ObjectID.GREEN_TEARS_6666).
+			build();
+	static final String WAIT_STRING = "Wait...";

```

Comment:
Wait for what? It wouldn't hurt to expand this to be clear what will make the overlay start working.